journal-of-water-sanitation-and-hygiene-for-development.csl

```diff
@@ -0,0 +1,604 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" default-locale="en-GB" demote-non-dropping-particle="never">
+  <info>
+    <title>Journal of Water Sanitation and Hygiene for Development</title>
+    <title-short>JWaSHDev</title-short>
+    <id>http://www.zotero.org/styles/journal-of-water-sanitation-and-hygiene-for-development</id>
+    <link href="http://www.zotero.org/styles/journal-of-water-sanitation-and-hygiene-for-development" rel="self" />
+    <link href="http://www.zotero.org/styles/apa-old-doi-prefix" rel="template" />
+    <link href="http://washdev.iwaponline.com/content/instructions-authors" rel="documentation" />
+    <author>
+      <name>Patrick O'Brien, PhD</name>
+      <email>obrienpat86@gmail.com</email>
+    </author>
+    <category citation-format="author-date" />
+    <category field="science" />
+    <eissn>2043-9083</eissn>
```

Comment:
It's due to not having found a proper source yet. If I don't find them on Wikipedia (which lists ISSN and eISSN) I look on Worldcat.org, but that only ever lists one number (which so far I have assumed was the ISSN). I'll make sure this doesn't happen anymore. Sorry.